accord-core/src/test/java/accord/topology/TopologyManagerTest.java

```diff
@@ -277,4 +304,280 @@ void truncateTopologyCantTruncateUnsyncedEpochs()
     {
 
     }
+
+    @Test
+    void removeRanges()
```

Comment:
BurnTest does not include remove range support, but TM does in this patch, so this test is still relevant.  The remove range in BurnTest has issues with multiple subsystems so was punted, but this subsystem was low effort so kept